src/cli/inputHelpers.ts

```diff
@@ -14,8 +14,26 @@
  *  limitations under the License.
  */
 
-declare module 'prompt' {
-  import prompt = require('prompt');
-  // TODO(andreban): Remove usage of any
-  export function get(schema: any): any; // eslint-disable-line @typescript-eslint/no-explicit-any
+import Color = require('color');
+import {isWebUri} from 'valid-url';
+
+export function validatePassword(input: string): boolean {
+  return input.length > 0;
```

Comment:
Sure we don't want something longer?

Reply:
I need to look at the requirements for keytool and replicate there. Doing on a following CL.